File: Client/src/app/services/model/team.ts
import { Guid } from 'guid-typescript';

export interface Team {

  id: string;
  name: string;
  user: string;
  giocatori: number;
  fantaMilioni: number;
  championshipFantaId: Guid;
}

export class TeamClass {
  constructor(
    public name: string,
    public user: string,
    public championshipFantaId: Guid
  ) { }
}

File: 2.1/fapi/Data/Entities/Team.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace fantacalcioApi.Data.Entities
{
    public class Team
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string User { get; set; }
        public List<Player> Giocatori { get; set; }
        public int FantaMilioni { get; set; }
        public string ChampionshipID { get; set; }
    }
}
